appcli/cli_builder.py

```diff
@@ -46,6 +47,22 @@ def create_cli(configuration: Configuration):
     ENV_VAR_GENERATED_CONFIG_DIR = f'{APP_NAME_UPPERCASE}_GENERATED_CONFIG_DIR'
     ENV_VAR_DATA_DIR = f'{APP_NAME_UPPERCASE}_DATA_DIR'
 
+    # --------------------------------------------------------------------------
+    # CREATE_CLI: LOGIC
+    # --------------------------------------------------------------------------
+
+    install_commands = InstallCli(configuration).commands
+    main_commands = MainCli(configuration).commands
+    configure_commands = ConfigureCli(configuration).commands
+    init_commands = InitCli(configuration).commands
+
+    all_commands = {
```

Comment:
probably clearer to call this `default_commands` since it is the in-built commands (i.e. excludes user defined commands)